### PR TITLE
[UIHC] Brand bar

### LINKED ISSUE #7670: UI Health Care Branding - CSS Updates to Logo
Ryan Potter, our UI Health Care Art Director has reviewed the recently pushed code for the UI Health Care Brand on the SIteNow platform and has a few CSS adjustment requests.

The crux of his review and requests can be summarized by a desire to see the IOWA portion of the logo not change, between UIOWA branded sites, and UI Health Care branded sites.

He has followed this practice on other mediums and applications, but strives wherever possible to keep the IOWA in the logo equally weighted. 

This does mean the IOWA HEALTH CARE logo ends up being taller than the IOWA logo, with the second line of text below it, but we do not diminish the visual weight IOWA is given in this approach, and he says it is significantly less jarring visually when moving from an IOWA to an UI HEalth Care branded item, or visa versa.

You can see in screenshots below:

On Site Now with a parent and site org name:
CURRENT IOWA:
Screenshot 2024-05-13 at 8 31 38 AM

CURREN UI HEALTH CARE:
Screenshot 2024-05-10 at 3 50 12 PM

If instead of shrinking the IOWA HEALTH CARE Logo to fit within the constraints, we adjust the height of the tab enough to allow for the additional height of HEALTH CARE, we would have the word IOWA remain the same size, and the tab extend tall enough to allow for the word health care below.

Example:
Screenshot 2024-05-10 at 3 51 37 PM

Ryan would like this same logic to be applied if possible to the site title/header design where the site name is in the gold bar.

Current Design Example:

Screenshot 2024-05-13 at 8 33 36 AM
Screenshot 2024-05-13 at 8 33 22 AM

If the word IOWA can remain unchanged between the two in sizing, and the black tab extends in height to allow for the word health care, that would be preferred.

Happy to share any thoughts back to Ryan if there are challenges or issues with this requested adjustment. 

```[tasklist]
### Tasks
- [ ] Update branding bar component in UIDS to use UIHC logo
- [ ] Remove custom UIHC CSS from https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/components/logo.scss
- [ ] Top menu will need to be adjusted https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/components/menus/top.scss
- [ ] Toggle nav will need to be adjusted https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/components/menus/toggle-nav.scss
- [ ] Search toggle will need to be adjusted https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/components/search-toggle.scss
```

### PR DESCRIPTION
**Pull request by bspeare**
Resolves https://github.com/uiowa/uiowa/issues/7670

# How to test

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli 
```

Test all header variations at mobile, tablet, and desktop breakpoints

## "UIHC Healthcare" theme

Toggle navigation

- Set https://sandbox.uiowa.ddev.site/admin/appearance to "toggle navigation style"
  - Set https://sandbox.uiowa.ddev.site/admin/appearance to "sticky" and run through the steps above again. 
 - Test all header variations at mobile, tablet, and desktop breakpoints
   - Test search toggle, menu toggle
 - Set "Site name display" to display inline and below for the toggle navigation
  - Scroll up/down the page and Test all header variations at mobile, tablet, and desktop breakpoints
  - Set "parent org" on https://sandbox.uiowa.ddev.site/admin/config/system/site-information and test the above steps again. 
  - Disable "top links" on https://sandbox.uiowa.ddev.site/admin/structure/menu/manage/top-links and test the above steps again. 
  - Test header with vertical/horizontal admin toolbar while logged **IN**
  - Test header while logged **OUT**

Horizontal/Mega navigation

- Set https://sandbox.uiowa.ddev.site/admin/appearance to "Horizontal navigation style"
- Set site name display with the IOWA bar and after that test with site name below IOWA Bar
- Test all header variations at mobile, tablet, and desktop breakpoints
  - Set "parent org" on https://sandbox.uiowa.ddev.site/admin/config/system/site-information and test the above steps again. 
  - Disable "top links" on https://sandbox.uiowa.ddev.site/admin/structure/menu/manage/top-links and test the above steps again. 
  - Test header with vertical/horizontal admin toolbar while logged **IN**
  - Test header while logged **OUT**

## "Iowa" theme

Test the above scenarios listed for "UIHC Healthcare" and look for any regressions. 

## "Regents" theme

Test the above scenarios listed for "UIHC Healthcare" and look for any regressions.

### CODE REVIEW

**Comment by joewhitsitt** [comment]
Pulling to test

**Comment by joewhitsitt** [comment]
@bspeare i am noticing that the following settings are hiding the breadcrumbs on an interior page (logged out)

Screenshot 2024-05-22 at 10 16 10 AM
Screenshot 2024-05-22 at 10 17 01 AM

UPDATE: I can't seem to reproduce this anymore. Ben mentioned it is an rarely used combo anyway.

File: docroot/themes/custom/uids_base/scss/components/brand-bar.scss [hunk 1]
```diff
@@ -74,3 +74,59 @@
...
+    @include breakpoint(md) {
+      margin-top: -17px;
+    }
+  }
+}
+
+.header-sticky.scroll-down .iowa-bar.iowa-bar--narrow {
+  .toggle-nav__bttn {
+    @include breakpoint(page-container) {
+      //top: 35px;
```
Comment (joewhitsitt) [inline docroot/themes/custom/uids_base/scss/components/brand-bar.scss]: should this be removed if the inner is commented out?

**Comment by joewhitsitt** [comment]
Probably not in-scope, but the regents with toggle and top-links seem adrift:

Screenshot 2024-05-22 at 11 11 20 AM

**Review by joewhitsitt — APPROVED** [review]
Might be a local env issue, but with some `drush cr` sprinkled in, this is all seems to be working as expected. It is starting to get tight between the health care logo and the logged-in task bar (e.g. View, Edit...). I've been trying to keep up with the moves from UIDS and have been checking back to see the one-to-one.  So at _this point in time_, I approve!

**Comment by bspeare** [comment]
Thanks for testing @joewhitsitt!

**Pull request merged**